doc/releases/changelog-dev.md

```diff
@@ -617,6 +617,9 @@
 * `qml.map_wires` no longer fails when mapping nested quantum tapes.
   [(#4901)](https://github.com/PennyLaneAI/pennylane/pull/4901)
 
+* `parity_transform`  is added for parity mapping of a fermionic Hamiltonian.
+   [(#4928)](https://github.com/PennyLaneAI/pennylane/pull/4928)
```

Comment:
Minor point, but example code here is always welcome.

Reply:
I am not sure that I understand, lines 620 and 621 are already there in the changelog. Do you mean we should add working example for parity transform here?

Reply:
> Do you mean we should add working example for parity transform here?

Yes, see some examples in Release 0.34.0 [changelog](https://github.com/PennyLaneAI/pennylane/blob/master/doc/releases/changelog-0.34.0.md).